### PR TITLE
symfonize frameworks index

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

**Review by alexarn — CHANGES_REQUESTED** [review]
Click on "nouveau tableau" => Click on "retour" => Access denied (index.php?page=planning/postes_cfg/index.php)

Click on "nouvelle ligne" goes to framework index (/framework?page=planning%2Fpostes_cfg%2Flignes_sep.php&modification=lignes_sep&cfg-type=lignes_sep)

After cloning a table => Access denied

After saving a new line => Acces denied

File templates/framework/edit.html.twig should not be in this pull request

**Comment by sillydebs** [comment]
> Click on "nouveau tableau" => Click on "retour" => Access denied (index.php?page=planning/postes_cfg/index.php)
> 
> Click on "nouvelle ligne" goes to framework index (/framework?page=planning%2Fpostes_cfg%2Flignes_sep.php&modification=lignes_sep&cfg-type=lignes_sep)
> 
> After cloning a table => Access denied
> 
> After saving a new line => Acces denied
> 
> File templates/framework/edit.html.twig should not be in this pull request

Thank you for your review. The code has been corrected as required.

**Comment by alexarn** [comment]
When i want to resore a table (framework), i select one in the list, my browser get freezed (and my laptop too)

File: public/planning/postes_cfg/ajax.infos.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 $id=filter_input(INPUT_GET, "id", FILTER_SANITIZE_NUMBER_INT);
 $CSRFToken = filter_input(INPUT_GET, "CSRFToken", FILTER_SANITIZE_STRING);
 $nombre=filter_input(INPUT_GET, "nombre", FILTER_SANITIZE_NUMBER_INT);
-$nom=filter_input(INPUT_GET, "nom", FILTER_SANITIZE_STRING);
```
Comment (alexarn) [inline public/planning/postes_cfg/ajax.infos.php:28]: why this change ? I'm not sure this file is used in the framework index page

File: public/admin/index.php [hunk 2]
```diff
@@ -50,7 +50,7 @@
 }
 
 if (in_array(22, $droits)) {
-    echo "<li><a href='index.php?page=planning/postes_cfg/index.php'>Les tableaux</a></li>\n";
+    echo "<li><a href='/framework'>Les tableaux</a></li>\n";
```
Comment (alexarn) [inline public/admin/index.php:53]: please, use asset

File: public/planning/postes_cfg/copie.php [hunk 3]
```diff
@@ -104,7 +104,7 @@
     }
 
     // Retour à  la page principale
-    echo "<script type='text/javaScript'>parent.location.href='index.php?page=planning/postes_cfg/index.php&cfg-type=horaires&numero=$numero2';</script>\n";
+    echo "<script type='text/javaScript'>parent.location.href='/framework?cfg-type=horaires&numero=$numero2';</script>\n";
```
Comment (alexarn) [inline public/planning/postes_cfg/copie.php:107]: please, use baseURL

File: public/planning/postes_cfg/groupes2.php [hunk 4]
```diff
@@ -28,4 +28,4 @@
 $t->CSRFToken = $CSRFToken;
 $t->update($post);
 
-echo "<script type='text/JavaScript'>location.href='index.php?page=planning/postes_cfg/index.php';</script>\n";
+echo "<script type='text/JavaScript'>location.href='/framework';</script>\n";
```
Comment (alexarn) [inline public/planning/postes_cfg/groupes2.php:31]: please, use baseURL

File: public/planning/postes_cfg/lignes_sep.php [hunk 5]
```diff
@@ -30,14 +30,14 @@
     $db=new db();
     $db->CSRFToken = $CSRFToken;
     $db->update("lignes", array("nom"=>$nom), array("id"=>$id));
-    echo "<script type='text/JavaScript'>document.location.href='index.php?page=planning/postes_cfg/index.php'</script>\n";
+    echo "<script type='text/JavaScript'>document.location.href='/framework'</script>\n";
```
Comment (alexarn) [inline public/planning/postes_cfg/lignes_sep.php:33]: Please use baseURL

File: public/planning/postes_cfg/lignes_sep.php [hunk 6]
```diff
@@ -30,14 +30,14 @@
...
     break;
 
   case "ajout2":
     $db=new db();
     $db->CSRFToken = $CSRFToken;
     $db->insert("lignes", array("nom"=>$nom));
-    echo "<script type='text/JavaScript'>document.location.href='index.php?page=planning/postes_cfg/index.php'</script>\n";
+    echo "<script type='text/JavaScript'>document.location.href='/framework'</script>\n";
```
Comment (alexarn) [inline public/planning/postes_cfg/lignes_sep.php:40]: Please use baseURL

File: public/planning/postes_cfg/modif.php [hunk 7]
```diff
@@ -60,7 +60,7 @@
 echo "<li><a href='#div_infos' id='infos'>Infos générales</a></li>\n";
 echo "<li><a href='#div_horaires' id='horaires'>Horaires</a></li>\n";
 echo "<li><a href='#div_lignes' id='lignes'>Lignes</a></li>\n";
-echo "<li class='ui-tab-cancel'><a href='index.php?page=planning/postes_cfg/index.php' >Retour</a></li>\n";
+echo "<li class='ui-tab-cancel'><a href='/framework' >Retour</a></li>\n";
```
Comment (alexarn) [inline public/planning/postes_cfg/modif.php:63]: Please use asset

**Review by alexarn — CHANGES_REQUESTED** [review]

File: public/planning/postes_cfg/ajax.infos.php (hunk 1 above)
Comment (sillydebs) [reply public/planning/postes_cfg/ajax.infos.php:28]: As far as I can remember, it was a matter of display while I as testing the whole. I will set this aside for now.

File: public/admin/index.php (hunk 2 above)
Comment (sillydebs) [reply public/admin/index.php:53]: As it not a twig template, shouldn't I use {$config['URL']} instead ?

File: public/planning/postes_cfg/ajax.infos.php (hunk 1 above)
Comment (sillydebs) [reply public/planning/postes_cfg/ajax.infos.php:28]: > As far as I can remember, it was a matter of display while I as testing the whole. I will set this aside for now.

I confirm my explanation : with the filter, quotes cannot be added.

**Comment by alexarn** [comment]
Replaced baseURL bay $config['URL'] in php files
Merged on master
